rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
+11) To reflect a consensus in a diverse community, the moderation team must embody cultural and viewpoint diversity, respecting members with varying societal views, provided they do not undermine equal community membership.
+12) We respect viewpoint diversity in cultural and political matters when selecting moderation team candidates.
+13) We understand that reasonable moderation is based on past events or future expectations, necessitating respect for historical records and transparency in controversial events.
+14) We uphold that true diversity necessitates exceptional tolerance, actively embracing not only the ideas we favor but, also, those we disagree with. This commitment to tolerance is fundamental in fostering a community where diverse viewpoints can coexist and enrich our collective experience.
+
+## Resolution to Adopt the *Memorandum*
+
+- Incorporate the *Memorandum* text into the moderation repository on GitHub, linking it prominently from the Code of Conduct and all relevant public information sources.
+- Implement one-year term limits for moderation team members, with a one-year hiatus before potential return. Apply this rule retroactively.
+- Align the current moderation team composition and the Code of Conduct with the *Memorandum* principles within three months of adopting this document.
+
+## Resolution to Establish an Appeals Council
+
+- Form an *Appeals Council* of three long-standing community members, reflecting cultural and viewpoint diversity.
+- Exclude current moderation team members from the *Appeals Council*, with former members eligible after a one-year hiatus.
+- Grant the council authority to override moderation team nominations, overturn decisions, and end moderation team memberships that conflict with the *Memorandum* principles.
+- Adopt a 2/3 majority rule for council decisions, disallowing abstentions to avoid indecision.
```

Comment:
How can someone recuse themself if they have a potential or apparent conflict of interest if they are barred from abstention?